xtool/env/env.go

```diff
@@ -20,29 +20,45 @@ import (
 	"os"
 	"os/exec"
 	"regexp"
-	"runtime"
 	"strings"
 )
 
+var (
+	reSubcmd = regexp.MustCompile(`\$\(([^)]+)\)`)
+	reFlag   = regexp.MustCompile(`[^ \t\n]+`)
+)
+
 func ExpandEnv(s string) string {
 	return expandEnvWithCmd(s)
 }
 
 func expandEnvWithCmd(s string) string {
-	re := regexp.MustCompile(`\$\(([^)]+)\)`)
-	expanded := re.ReplaceAllStringFunc(s, func(m string) string {
-		cmd := re.FindStringSubmatch(m)[1]
+	expanded := reSubcmd.ReplaceAllStringFunc(s, func(m string) string {
+		subcmd := s[2 : len(s)-1]
+
+		args := parseSubcmd(subcmd)
+
+		cmd := args[0]
+
+		if cmd != "pkg-config" && cmd != "llvm-config" {
+			// TODO: log in verbose mode
```

Comment:
This error message should print even not in verbose.
And `// TODO(udi):` is recommended by Go.

Reply:
Should we print it only or panic?

Reply:
print to stderr (don't panic)